PsfLauncher/main.cpp

```diff
@@ -304,7 +315,89 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
     }
 }
+

```

Comment:
Does PSF need to have any fixups applied for trace logging to work?

Reply:
not really, but if there is fix up and trace logging is enabled then the data will be collected

---
In reply to: [287523430](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/41#discussion_r287523430) [](ancestors = 287523430)